src/util/DataResolver.js

```diff
@@ -88,7 +88,7 @@ class DataResolver {
 
     if (typeof resource === 'string') {
       if (/^https?:\/\//.test(resource)) {
-        return snekfetch.get(resource).then(res => res.body instanceof Buffer ? res.body : Buffer.from(res.text));
+        return snekfetch.get(resource).then(res => res.body instanceof Buffer ? res.body : res.raw);
```

Comment:
This check is useless, `res.raw` is always a Buffer instance.

Reply:
Good catch